vue/src/App.vue

```diff
@@ -1,19 +1,24 @@
 <template>
   <div id="app">
-    <img alt="Vue logo" src="https://vuejs.org/images/logo.png">
-    <HelloWorld msg="Welcome to Your Vue.js App"/>
+    <img
+      alt="Vue logo"
+      width="200"
+      height="200"
```

Comment:
Longstanding issue with our Vue starter template: logo looks too big. Because:

- In the original project generated by `@vue/cli`, there is a `src/assets/logo.png` file. Because storing binary files in StackBlitz is a paid feature, we modified that to link to https://vuejs.org/images/logo.png
- The `src/assets/logo.png` file was 200x200, but the image we link to is 400x400, and ends up looking way too big in the preview iframe in our classic editor.

Other options that don't require hotlinking to the vuejs.org site:

1. Include a SVG logo as `src/assets/logo.svg`. Problem: EngineBlock doesn't seem to support it, at least not in the `vue-cli` preset.
2. Include the logo as inline SVG in the template. That one is doable, and the Vue logo can be tiny as SVG:

```svg
<svg viewBox="0 0 16 16" width="200" height="200">
  <path d="M9.85 1.07L8 4.27l-1.85-3.2H0l8 13.86 8-13.86H9.85z" fill="#41B883"/>
  <path d="M9.85 1.07L8 4.27l-1.85-3.2H3.2L8 9.4l4.8-8.32H9.85z" fill="#34495E"/>
</svg>
```

For now I'm keeping the hotlinked PNG, but if we want to go with inline SVG (maybe even as a `VueLogo.vue` component that takes a `size` prop?) that works too.